test/components/test_alumni.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react/addons";
+let { TestUtils } = React.addons;
+
+import Alumni from "../../src/components/alumni";
+
+
+describe("Alumni Component", function() {
+  it("should render the correct title", function() {
+    let component = TestUtils.renderIntoDocument(
+      <Alumni />
+    );
+
+    let heading = TestUtils.findRenderedDOMComponentWithTag(component, "h1");
+
+    expect(heading.getDOMNode().textContent).to.equal('Alumni');
```

Comment:
Let's use `findDOMNode`.
https://facebook.github.io/react/docs/component-api.html#getdomnode
https://facebook.github.io/react/docs/working-with-the-browser.html#refs-and-finddomnode